### PR TITLE
Node/SDK: Add neutron support via IBC

### PR DESCRIPTION
**Pull request by bruce-riley**

### CODE REVIEW

File: sdk/js/src/utils/consts.ts [hunk 1]
```diff
@@ -513,6 +520,11 @@ const TESTNET = {
     token_bridge: undefined,
     nft_bridge: undefined,
   },
+  neutron: {
+    core: undefined,
```
Comment (nik-suri) [inline sdk/js/src/utils/consts.ts:524]: Same as above - the `wormhole-ibc` contract is deployed on Neutron testnet with address `neutron1rhqgnak5z0pla5sgqq7ykqh8glckk4h67fe0hgjfcyulx2nqa0yqaswtnp`

File: sdk/js/src/utils/consts.ts [hunk 2]
```diff
@@ -308,6 +310,11 @@ const MAINNET = {
...
```
Comment (nik-suri) [inline sdk/js/src/utils/consts.ts:314]: The wormhole-ibc contract is deployed on Neutron mainnet with address neutron16rerygcpahqcxx5t8vjla46ym8ccn7xz7rtc6ju5ujcd36cmc7zs9zrunh

**Review by evan-gray — APPROVED** [review]

**Comment by bruce-riley** [comment]
@nik-suri give me those addresses.

**Review by SEJeff — APPROVED** [review]

**Comment by evan-gray** [comment]
> I didn't know how to double check the core contract addresses. But reviewed the rest.

I checked the explorers
https://www.mintscan.io/neutron/wasm/contract/neutron16rerygcpahqcxx5t8vjla46ym8ccn7xz7rtc6ju5ujcd36cmc7zs9zrunh
https://www.mintscan.io/neutron-testnet/wasm/contract/neutron1rhqgnak5z0pla5sgqq7ykqh8glckk4h67fe0hgjfcyulx2nqa0yqaswtnp

**Pull request merged**